test/functional/wallet_send.py

```diff
@@ -133,6 +139,11 @@ def test_send(self, from_wallet, to_wallet=None, amount=None, data=None,
             assert not "txid" in res
             assert "psbt" in res
 
+        if include_unsafe:
+            from_balance = from_wallet.getbalances()["mine"]["untrusted_pending"]
+        else:
+            from_balance = from_wallet.getbalance()
```

Comment:
I'm not sure that change should be done in this PR.
It may be worth sweeping all uses of `getbalance()` to replace them with `getbalances()["mine"]["trusted"]` in a separate PR, don't you think?
I volunteer to make that follow-up PR right after this one if that sounds good to you.

Reply:
it's new code, if you don't mind conforming to the newer standard for new code makes any future PR to "sweep" it up smaller?

Reply:
Fair enough, and this is quite simple, done in https://github.com/bitcoin/bitcoin/pull/21359/commits/1bb252669dc5e8feb41e4975d609108eb28c8667